components/media/PreviewMediaResolver.vue

```diff
@@ -33,7 +34,9 @@ const fetchMimeType = async () => {
     return
   }
 
-  const nftMetadata = await get<NFTMetadata>(props.metadata)
+  const nftMetadata = (await processSingleMetadata(
+    props.metadata
+  )) as NFTMetadata
```

Comment:
```suggestion
  const nftMetadata = await processSingleMetadata<NFTMetadata>(props.metadata)
```

Reply:
aahh, good call to summon you to this PR XD
will check